### PR TITLE
[RFR] Feature: Build Automation

### PR DESCRIPTION
**Pull request by verpixelt**
- [x] Create Scss compile task (@lewiscowper)
  - [x] Write documentation (@lewiscowper)
- [x] Create local server task (@lewiscowper)
  - [x] Write documentation (@lewiscowper)
- [x] Create linting task (@busyxin)
  - [x] Write documentation (@lewiscowper)
- [x] Create visual regression testing task (Moved to #39)
  - [x] Write documentation (Moved to #39)
- [x] Create documentation task (@lewiscowper)
  - [x] Write documentation (@lewiscowper)
- [x] Extend local server task with live reload and watch tasks (@lewiscowper)
  - [x] Write documentation (@lewiscowper)
- [x] Create production CSS tasks (minify, prefix) (@busyxin)
  - [x] Write documentation (@lewiscowper)

Also we have a slight readability issue when using parallelshell, and it's another step for people to understand. The `npm-run-all` package will greatly improve how easy it is to comprehend what our scripts are doing.
- [x] Remove parallelshell, refactor all scripts with npm-run-all

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <script src="http://localhost:9091"></script>
+</head>
+<body>
+  <h1>Hello World</h1>
+</body>
```
Comment (Charlotteis) [inline dist/index.html]: Need a newline after this line :)

File: dist/index.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
```
Comment (varjmes) [inline dist/index.html]: `<html lang="en">`

> "Declaring a language attribute on the html element enables a screen reader to read out the text with correct pronunciation." http://a11yproject.com/checklist.html

Comment (varjmes) [inline dist/index.html]: Also could do with a `<title>` in the header here.

Though these seem like little things, if we get them sorted from the beginning, it makes everything nice and clean and full of awesome from the start.

**Comment by lewiscowper** [comment]
We'll merge this as is, and open a new PR for the visual regression tests. See #38 for more discussion on this. It's a way more complex topic than previously estimated.

**Pull request merged**